database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/action/DBRun.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.plugin.db.batch.action;
+
+import co.cask.hydrator.plugin.DBManager;
+import org.slf4j.Logger;
+
+import java.sql.Connection;
+import java.sql.Driver;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
+ * Class used by database action plugins to run database commands
+ */
+public class DBRun {
+  private final QueryConfig config;
+  private final Class<? extends Driver> driverClass;
+  private final Logger log;
```

Comment:
don't pass in the logger. Logger should pretty much always be a private static final field of whatever class its in.